src/component/common/ConditionallyRender/ConditionallyRender.tsx

```diff
@@ -1,5 +1,5 @@
 interface IConditionallyRenderProps {
-    condition: boolean;
+    condition: boolean | string | undefined;
```

Comment:
I find it a bit confusing that the condition can be `boolean | string | undefined;`

Reply:
Agreed, that is better. Fixed!

Reply:
Yeah, I think we could just have this as boolean and do the conversion. Makes it easier to reason about.